os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -182,12 +182,12 @@ rtw_result_t app_scan_result_handler(rtw_scan_handler_result_t *malloced_scan_re
 /*

```

Comment:
May be you can add the assert logs as well in the commit description.

Reply:
@gexuyan You should describe why the previous way to create threads cause assertion in the commit description.

Reply:
[tp1x.log](https://github.com/Samsung/TizenRT/files/6033631/tp1x.log)
Dear @thapav Sure, the original assert log is as attachment above. I will also add in the commit description.
Dear @davidfather Because linkup_handler and linkdown_handler are called many times during the stress test. The previous way to create threads may result in many cbk_handler threads be created at same time. The new way to create thread ensures that only one cbk_handler is created because there's only one tid for cbk_handler thread. I will also add this description in the commit description. 
Thank you~